hr/overview/employee_form.rst

```diff
@@ -0,0 +1,123 @@
+=============
+Employee Form
+=============
+
+
+Create an employee Form
+=======================
+
+You can **Create** an employee from the employee dashboard or directly from a department.
+
+
+.. image:: media/create-employee.png
+   :align: center
+   :alt: How to create an employee
+
+If you have an existing list of employees, you can *Import* them in a CSV or Excel file, to batch import your employees,
+simply click on the *import* button on the dashboard.
+
+.. note::
+   Need help ? `how to import your contacts <https://www.odoo.com/slides/slide/contacts-and-importing-data-609?fullscreen=1>`_.
+
+Complete employee form
+======================
+
+The employee form allows you to add tags, a picture and edit all the general information about your employee.
+You can also create and edit departments and a manager for your employee.
+
+
+.. image:: media/employee-form.png
+   :align: center
+   :alt: Empty employee form
+
+Print
+=====
+
+From the employee form, *Print Badge*, *Time Off Summaries* and *Individual Accounts* for each employee.
+
+.. image:: media/print-actions.png
+   :align: center
+   :alt: Print actions available on the employee form
+
+Action
+======
+
+You can also directly *Archive*, *Delete*, *Duplicate* or *Generate tax forms* and *Attestations* from the employee
+form.
+
+.. note::
+
+   281.10 and 281.45 are tax forms for belgian localisations.
+
+.. image:: media/actions.png
+   :align: center
+   :alt: Actions available on the employee form
+
+Employee Skill management
+=========================
+In the *Resumé* tab in the employee form, you can keep track of the experiences and skills of your employees. Create and
+edit skills and experience by clicking on *Edit* →  **Add**.
+You can also simply delete lines by clicking on the bin icon
+
+.. image:: media/create-skills.png
+   :align: center
+   :alt: Create skills for your employees
+
+Add Skills
+----------
+When creating a Skill you can choose the *Skill Type, the Skill level, who will determine your employee progress.
+Directly create and edit different *Skill Types*, *Skills* and *Skill Levels* from this form
+
+
+.. image:: media/add-experience.png
+   :align: center
+   :alt: Filling skill forms for your employees
+
+Add Experience
+--------------
+When adding an experience, you can pick different categories of experience.
```

Comment:
```suggestion

When adding an experience, you can pick different categories of experience.
```
This paragraph could be improved by adding a few words of what the form shows once you add an employee experience?!

The image emphasizes the Education title, but nothing is mention in the text?!